README.md

```diff
@@ -101,13 +101,20 @@ clojure -Sdeps '{:deps {cljfmt {:mvn/version "0.9.0"}}}' \
 
 ## Configuration
 
+### Leiningen
 You can configure lein-cljfmt by adding a `:cljfmt` map to your
 project:
 
 ```clojure
 :cljfmt {}
 ```
 
+### Command line
+You can configure `cljfmt` by placing a `.cljfmt.edn` file in the directory
```

Comment:
Lines should be 72 characters or fewer the the README.

Reply:
done